serving/samples/helloworld-java-spark/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.example.helloworld;
```

Comment:
Why do we need this example file?

Reply:
What do you mean?  I have set it up as per the Spring boot app.